framer/Version.coffee.template

```diff
@@ -0,0 +1,7 @@
+# This gets automatically populated bij gulp
+
+exports.date = <%= date %>
```

Comment:
It'll be impossible to rebuild an exact version replica when date is always build time's 'now'. So two builds functionally equivalent won't have the same file hash. Not sure if that matters though; might be handy to have the date for future use.